ros/processor/insights_engine_result_consumer.py

```diff
@@ -234,3 +236,10 @@ def trigger_notification(
                     self.prefix, inventory_id, account.account, account.id, account.org_id
                 )
                 new_suggestion_event(host, platform_metadata, system_previous_state, system_current_state, producer)
+
+
+if __name__ == "__main__":
+    start_http_server(int(METRICS_PORT))
+    commence_cw_log_streaming('ros-processor')
```

Comment:
likewise, can we have different logging stream?

Reply:
Done

Reply:
If we have different logging streams for engine and events, IMO it will be a bit more work to correlate if any _x_ system has been updated after creation, thoughts @upadhyeammit @patilsuraj767 ?

Reply:
I am still exploring on logging, so if keeping is same makes sense and better approach then we should go with that!

Reply:
@patilsuraj767 Could you post a poll on this?

Reply:
Changes pushed as per internal poll results